src/SampleWebApp/Controllers/BooksMvcController.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+
+using SampleWebApp.Data;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Controllers
+{
+    public class BooksMvcController : Controller
+    {
+        private BookDataContext _context;
+
+        public BooksMvcController(BookDataContext context)
+        {
+            _context = context;    
+        }
+
+        // GET: BooksMvc
+        public IActionResult Index()
+        {
+            var bookDataContext = _context.Book.Include(b => b.Author);
```

Comment:
I would use the DbSet property name, which in this example is `Book`, but in most real apps would be `Books` (BTW that is what we do for naming tables starting in RC2).